core/src/main/java/org/apache/iceberg/MetadataColumns.java

```diff
@@ -38,6 +38,12 @@ private MetadataColumns() {
       Integer.MAX_VALUE - 2, "_pos", Types.LongType.get(), "Ordinal position of a row in the source data file");
   public static final NestedField IS_DELETED = NestedField.required(
       Integer.MAX_VALUE - 3, "_deleted", Types.BooleanType.get(), "Whether the row has been deleted");
+  public static final NestedField SPEC_ID = NestedField.required(
+      Integer.MAX_VALUE - 4, "_spec_id", Types.IntegerType.get(), "Spec ID to which a row belongs to");
```

Comment:
Nit: no need for "to" at the end of the doc because it already uses "to which".

Reply:
I think I would also rephrase this because the row doesn't belong to a spec. How about "Spec ID used to track the data file containing a row"?

Reply:
Changed.